$OpenBSD: patch-include_ewmh_h,v 1.1 2011/12/01 16:00:01 dcoppa Exp $

Do not set the _NET_WM_WORKAREA hint: fix incorrect transient
geometry with wxwidgets
(upstream git commit d25e77ce7597bf0736c610a04185c8c27694c890)

--- include/ewmh.h.orig	Fri Nov 11 23:41:20 2011
+++ include/ewmh.h	Thu Dec  1 14:07:37 2011
@@ -29,16 +29,6 @@ void ewmh_update_current_desktop();
 void ewmh_update_active_window(xcb_window_t window);
 
 /**
- * Updates the workarea for each desktop.
- *
- * EWMH: Contains a geometry for each desktop. These geometries specify an area
- * that is completely contained within the viewport. Work area SHOULD be used by
- * desktop applications to place desktop icons appropriately.
- *
- */
-void ewmh_update_workarea();
-
-/**
  * Updates the _NET_CLIENT_LIST_STACKING hint. Necessary to move tabs in
  * Chromium correctly.
  *
